config/locales/proper.en.yml

```diff
@@ -242,9 +254,11 @@ en:
         get_cost_estimate: "Get a Cost Estimate"
         save_and_continue: "Save and Continue →"
         submit: "Submit Request →"
+        submit_request: "Submit Request"
```

Comment:
This value is pulled from the above `submit` key

https://github.com/sparc-request/sparc-request/blob/v3.5.0/app/views/service_requests/navigation/_footer.html.haml